Dark Theme/DarkTheme.js

```diff
@@ -73,15 +78,38 @@ tau
             addCSSRule('.tau-board-view .tau-label__velocity, .tau-board-view .tau-label__effort', 'color: #A1A7B3 !important;');
             addCSSRule('.tau-paging-info', 'color: #fff !important; font-weight: 400 !important');
             addCSSRule('.boardsettings-filter__contener:after', 'background: #292c33 !important; box-shadow: none !important;');
-            addCSSRule('.tau-cellholder .tau-label:after, .tau-x-header .tau-label:after, .tau-backlog-header .tau-label:after', 'box-shadow: none !important;')
-        };
+            addCSSRule('.tau-cellholder .tau-label:after, .tau-x-header .tau-label:after, .tau-backlog-header .tau-label:after', 'box-shadow: none !important;');
+        }};
         topMenu.addItem('TV').onClick(function() {
             var $st = $(document).find('#tv-style');
+            var d = new Date();
+            d.setTime(d.getTime() + (365*24*60*60*1000));
+            var expires = "expires="+ d.toUTCString();
```

Comment:
Suggest to add `secure` and `sameSite` options to the cookie for privacy. These prevents transferring such cookie over plain http when possible and do not send cookie to the external sites:

```
var expires = ";expires="+ d.toUTCString();
// If using https proto, set secure cookie to protect against stealing via regular http: proto calls.
var isHttps = window.location.protocol === 'https:';
// Cookie to only be transmitted over secure protocol as https.
var secure = isHttps ? ';secure' : '';
// SameSite prevents the browser from sending this cookie along with cross-site requests.
// The strict value will prevent the cookie from being sent by the browser to the target
// site in all cross-site browsing context, even when following a regular link.
var sameSite = ";samesite=strict";                

if ($st.length) {
    document.cookie = "TPtheme=Light" + expires + secure + sameSite;
    $st.remove();
} else {
    document.cookie = "TPtheme=Dark" + expires + secure + sameSite;
    init();
}  
``` 

See [MDN > cookie](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#Write_a_new_cookie) for details.

Reply:
Good suggestions, thanks.